gandi/cli/commands/config.py

```diff
@@ -0,0 +1,67 @@
+""" Configuration namespace commands. """
+
+import sys
+import os
+import subprocess
+
+import click
+
+from gandi.cli.core.cli import cli
+from gandi.cli.core.params import pass_gandi
+
+
+@cli.command()
+@click.option('-g', help='Get from global configuration only'
+                         '(default=env -> local -> global).',
+              is_flag=True, default=False)
+@click.argument('key')
+@pass_gandi
+def get(gandi, g, key):
+    """Display value of a given config key."""
+    val = gandi.get(key=key, global_=g)
+    if not val:
+        gandi.echo("No value found.")
+        sys.exit(1)
+    gandi.echo(val)
+
+
+@cli.command()
+@click.option('-g', help='Edit global configuration (default=local).',
+              is_flag=True, default=False)
+@click.argument('key')
+@click.argument('value')
+@pass_gandi
+def set(gandi, g, key, value):
+    """Update or create config key/value."""
+    gandi.configure(global_=g, key=key, val=value)
+
+
+@cli.command()
+@click.option('-g', help='Edit global configuration (default=local).',
+              is_flag=True, default=False)
+@pass_gandi
+def edit(gandi, g):
+    """Edit config file with prefered text editor"""
+    config_file = gandi.home_config if g else gandi.local_config
+    path = os.path.expanduser(config_file)
+    editor = gandi.get('editor', 'vi')
```

Comment:
gandi.get('editor') already checks the environment variable.
Are you saying you'd prefer asking the user for his prefered editor rather than using `vi` as default?

(See previous comments on outdated diff for discussion about this)